crates/symbolicator-proguard/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "symbolicator-proguard"
+publish = false
+version = "24.1.2"
+authors = ["Sentry <hello@getsentry.com>"]
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+futures = "0.3.12"
+proguard = "5.4.0"
+reqwest = { version = "0.11.0", features = [
+    "gzip",
+    "brotli",
+    "deflate",
+    "json",
+    "stream",
+    "trust-dns",
+] }
```

Comment:
I’m surprised we need this?

Reply:
You're right, that's a copy-paste mistake